### PR TITLE
Fixing the weird API response causing graphql errors

### PR DESCRIPTION
**Pull request by novitae**

### CODE REVIEW

File: instagram_scraper/app.py [hunk 1]
```diff
@@ -605,7 +605,11 @@ def augment_node(self, node):
                         urls += self.augment_node(carousel_item['node'])['urls']
                     node['urls'] = urls
                 else:
-                    node['urls'] = [self.get_original_image(details['display_url'])]
+                    try:
+                        node['urls'] = [self.get_original_image(details['display_url'])]
+                    except (TypeError, KeyError):
+                        pt1 = details['carousel_media']
+                        node['urls'] = [pt1[e]['image_versions2']['candidates'][0]['url'] for e in range(len(pt1))]
```
Comment (AlexNik) [inline instagram_scraper/app.py:612]: `node['urls'] = [carousel_media['image_versions2']['candidates'][0]['url'] for carousel_media in details['carousel_media'])]`

Comment (AlexNik) [reply instagram_scraper/app.py:612]: Why 0? Is it the highest resolution by default?

**Review by AlexNik — CHANGES_REQUESTED** [review]

File: instagram_scraper/app.py (hunk 1 above)
Comment (novitae) [reply instagram_scraper/app.py:612]: idk why but the link of the medias is contained in a list containing only one element, theses links (for different sizes of pics)

Comment (AlexNik) [reply instagram_scraper/app.py:612]: @novitae Could you fix the code review and i'll merge it.

Comment (AlexNik) [reply instagram_scraper/app.py:612]: @novitae ?

Comment (novitae) [reply instagram_scraper/app.py:612]: Sorry i deleted my work so i can't tell you more details. If you want i can send you the api response that causes the graphql error, so you can see by yourself why i am filling these dictionary paths

Comment (AlexNik) [reply instagram_scraper/app.py:612]: @novitae Could you fix the code review and i'll merge it.
Change 611, 6112 lines to
```
node['urls'] = [carousel_media['image_versions2']['candidates'][0]['url'] for carousel_media in details['carousel_media'])]
```

**Pull request closed without merging**